ci/travis/build-docker-images.py

```diff
@@ -0,0 +1,260 @@
+        for i in range(2):
+            output = DOCKER_CLIENT.api.build(
+                path=os.path.join(_get_root_dir(), "docker", image_name),
+                tag=tagged_name,
+                nocache=True,
+                buildargs=build_args)
+
+            full_output = ""
+            try:
+                start = datetime.datetime.now()
+                current_iter = start
+                for line in output:
+                    # print(line)
+                    if datetime.datetime.now(
+                    ) - current_iter >= datetime.timedelta(minutes=5):
+                        current_iter = datetime.datetime.now()
+                        elapsed = datetime.datetime.now() - start
+                        print(f"Still building {tagged_name} after "
+                              f"{elapsed.seconds} seconds")
+                    full_output += line.decode("utf-8")
+            except Exception as e:
+                print(f"FAILURE with error {e}")
+
+            if len(DOCKER_CLIENT.api.images(tagged_name)) == 0:
+                print(f"ERROR building: {tagged_name} & error below:")
+                print(full_output)
+                if (i == 1):
+                    raise Exception("FAILED TO BUILD IMAGE")
+                print("TRYING AGAIN")
+            else:
+                break
+
+        print("BUILT: ", tagged_name)
+        built_images.append(tagged_name)
+    return built_images
+
+
+def copy_wheels():
+    root_dir = _get_root_dir()
+    wheel = _get_wheel_name()
+    source = os.path.join(root_dir, ".whl", wheel)
+    ray_dst = os.path.join(root_dir, "docker/ray/.whl/")
+    ray_dep_dst = os.path.join(root_dir, "docker/ray-deps/.whl/")
+    os.makedirs(ray_dst, exist_ok=True)
+    shutil.copy(source, ray_dst)
+    os.makedirs(ray_dep_dst, exist_ok=True)
+    shutil.copy(source, ray_dep_dst)
+
+
+def build_or_pull_base_images(is_docker_affected: bool) -> List[str]:
+    """Returns images to tag and build"""
+    _ = DOCKER_CLIENT.api.pull(
+        repository="rayproject/base-deps", tag="nightly")
+
+    age = DOCKER_CLIENT.api.inspect_image("rayproject/base-deps:nightly")[
+        "Created"]
+    short_date = datetime.datetime.strptime(age.split("T")[0], "%Y-%m-%d")
+    is_stale = (
+        datetime.datetime.now() - short_date) > datetime.timedelta(days=14)
+
+    if is_stale or is_docker_affected or _release_build():
+        for image in ["base-deps", "ray-deps"]:
+            _build_cpu_gpu_images(image)
+        return True
+    else:
+        print("Just pulling images!")
+        _ = DOCKER_CLIENT.api.pull(
+            repository="rayproject/base-deps", tag="nightly-cpu")
+        _ = DOCKER_CLIENT.api.pull(
+            repository="rayproject/base-deps", tag="nightly-gpu")
+
+        _ = DOCKER_CLIENT.api.pull(
+            repository="rayproject/ray-deps", tag="nightly-gpu")
+        _ = DOCKER_CLIENT.api.pull(
+            repository="rayproject/ray-deps", tag="nightly-cpu")
```

Comment:
nit: is there any reason to use `_ = ` instead of just calling the function without assigning to a variable?

Reply:
To avoid printing the output of the call.